pkgs/by-name/lo/lofreq/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  autoreconfHook,
+  htslib,
+  python3,
+  zlib,
+
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lofreq";
+  version = "2.1.5";
+
+  src = fetchFromGitHub {
+    owner = "CSB5";
+    repo = pname;
+    tag = "v${version}";
+    hash = "sha256-dA2B3dCD6LbJ1hS87XVR442GJc2B4WhcO1PJV4fNWmM=";
+
+    postFetch = ''
+      # just .tar.gz'd tag archives -- too large for no use with keeping
+      rm -rf dist
+    '';
```

Comment:
not sure to understand what we are doing here

Reply:
dist contains the source code of every single tag -- would like to avoid putting that into the nix store